File: python/resistor-color-duo/resistor_color_duo.py
def value(colors):
    res = ''
    band_colors = [
        'black',
        'brown',
        'red',
        'orange',
        'yellow',
        'green',
        'blue',
        'violet',
        'grey',
        'white'
    ]
    for color in colors[:2]:
        index = str(band_colors.index(color))
        res += index
    return int(res)

print(value(["blue", "grey"]))

File: javascript/resistor-color-duo/resistor-color-duo.js
//
// This is only a SKELETON file for the 'Resistor Color Duo' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

const COLORS = [
    "black",
    "brown",
    "red",
    "orange",
    "yellow",
    "green",
    "blue",
    "violet",
    "grey",
    "white",
];

export const decodedValue = (colors) => {
    const color1 = COLORS.indexOf(colors[0]);
    const color2 = COLORS.indexOf(colors[1]);
    const result = String(color1) + String(color2);
    return parseInt(result);
};

console.log(decodedValue(["brown", "black"]));
